rust/Cargo.toml

```diff
@@ -46,44 +46,125 @@ readonly = "0.2"
 bitvec = "1.0"
 
 # A strict version requirement is necessary to keep the serialization format of LazyFrames stable.
-# If you change the version of Polars, 
-# be sure to also change the Python package version requirement and test binary compatibility.
-polars = { version = "=0.44.2", features = [
-    "lazy",
-    "csv",
-    "dtype-struct",
-    "dtype-array",
-    "dtype-date",
-    "dtype-datetime",
-    "dtype-time",
-    "parquet",
-    "serde-lazy",
-    "streaming",
-    "round_series",
-    "rank",
-    "polars-ops",
-    "abs",
+#   If you change the version of Polars, 
+#   be sure to also change the Python package version requirement and test binary compatibility.
+# The features track the features enabled in the Rust build for the Polars Python crate.
+#   Having matching features is necessary for cross-binary DSL compatibility.
```

Comment:
- Perhaps just a blank line (instead of indenting) to distinguish sections?
- "also change the Python package version requirement": I think there are at least four files that need to be kept in sync? Maybe add a test that just reads these files and checks?
- Having the same boilerplate text in all these files also adds up. Maybe one place has the authoritative description, and the others just point to it?